Upgrade package

strangerkir opened the pull request:
Port virtual environment from [dhii/php-project](https://github.com/Dhii/php-project):
* Use `build` and `test` images;
* xdebug version comes from variable in .env file
* Dockerfile_php renamed to Dockerfile to follow common practice

strangerkir committed:
port virtual environment from dhii/php-project

* Use `build` and `test` images;
* xdebug version comes from variable in .env file
* Dockerfile_php renamed to Dockerfile to follow common practice

strangerkir committed:
update required PHP version to ^7.2 | ^8.0

strangerkir committed:
update dependencies

strangerkir committed:
raise required phpunit version to ^8.0 | ^9.0

strangerkir committed:
update PHP language level in PhpStorm config

strangerkir committed:
make tests pass again

strangerkir committed:
Add PHP 8.0 to the list of versions to run CI

strangerkir committed:
Don't run CI on unsupported outdated PHP versions

strangerkir committed:
Fix PHPUnit config: remove nonexistent testsuite

strangerkir committed:
Fix docker-compose file

strangerkir committed:
Set default PHP test version as lowest supported

strangerkir committed:
Update PhpStorm config

XedinUnknown commented on .env.example:
```diff
@@ -1,2 +1,11 @@
+PHP_BUILD_VERSION=7.2
+PHP_TEST_VERSION=7.2
+
+XDEBUG_VERSION=3.0.4
```
Does this actually work for you? I still use xDebug 2.x everywhere, because I am sure it works. I'm very pro using 3.x, but gotta make sure it works for a Win setup first.

XedinUnknown replied:
Scratch tihis. It works for me.

XedinUnknown committed:
Bring environment further up to date

- Remove extra meta files;
- Update interpreter configuration;
- Update and clean tooling configuration;
- Update PHPStorm configuration;
- Tweak what goes into dist archive;
- Rename tests dir to `tests`;
- Update deps.
